services-custom/s3-transfermanager/src/main/java/software/amazon/awssdk/custom/s3/transfer/internal/S3CrtUtils.java

```diff
@@ -48,6 +49,8 @@
 
 @SdkInternalApi
 public final class S3CrtUtils {
+    private static final String HEADER_USER_AGENT = "User-Agent";
+    private static final String USER_AGENT_STRING = UserAgentUtils.getUserAgent() + " ft/s3-transfer";
```

Comment:
UserAgentUtils is currently an internal API.

Reply:
As discussed offline, changed it to protected API